### DISCUSSION TITLE
PaddleOCR/applications/%E6%89%8B%E5%86%99%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13830] [Q&A]
# PaddleOCR/applications/%E6%89%8B%E5%86%99%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/applications/%E6%89%8B%E5%86%99%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB.html

**Comment by Abird-py** [comment]
请问PP-OCRv4的文档什么时候可以推出呀

**Reply by UserWangZz** [reply]
请问是对v4的使用有什么问题吗？文档目前都集中在该链接下：https://paddlepaddle.github.io/PaddleOCR/

**Reply by Abird-py** [reply]
我看了issue #11200 我跟这个是同样的问题 底下没有解决方案

**Reply by Abird-py** [reply]
找到了一个issue下提供的yml配置文件同样的数据集v3的准确率53.4 v4模型17.7.....

**Reply by GreatV** [reply]
@Abird-py  可以试试我们的main分支，这些问题应该已经修复。

**Reply by Abird-py** [reply]
ch_PP-OCRv4_rec.yml么

**Reply by Abird-py** [reply]
V4的话需要像文档里一样修改对应的参数么
